pkg/cmd/container/run_runtime.go

```diff
@@ -47,6 +48,9 @@ func generateRuntimeCOpts(cgroupManager, runtimeStr string) ([]containerd.NewCon
 				}
 				runtimeOpts = nil
 			}
+		} else if filepath.IsAbs(runtimeStr) {
+			// runtimeStr is absolute path to runtime binary
```

Comment:
Runc-compatible binary should be assumed unless the binary name is composed of reversed domains

Reply:
We can also consider “CSV” values like “type=runc,path=foo”, “type=containerd,name=com.example.foo.v1”, “type=containerd,path=containerd-shim-foo-v1”